openapi/components/schemas/Plans/Formulas/fixed-fee.yaml

```
@@ -6,9 +6,12 @@ allOf:
     properties:
       price:
         description: |-
-          For the very simple price when it's fixed and does not depend on the quantity chosen by customer.
+          Use this formula to charge for subscriptions that involve the same price,
```

nitpick: It looks like you're trying to avoid using 'fixed price', but I think this is actually more clear:
```suggestion
          Use this formula to charge for subscriptions that have a fixed price,
```